cli/lib/kontena/cli/stacks/registry/show_command.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../common'
+
+module Kontena::Cli::Stacks::Registry
+  class ShowCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::Stacks::Common
+    include Kontena::Cli::Stacks::Common::StackNameParam
+
+    banner "Shows information about a stack on the stacks registry"
+
+    option ['-v', '--versions'], :flag, "Only list available versions"
```

Comment:
It's just a case of adding an if. I can try, I'm on windows and don't have my terminal magic.